.github/workflows/main.yml

```diff
@@ -217,16 +223,17 @@ jobs:
             Library-
 
       # Configure test runner
-      - uses: webbertakken/unity-test-runner@v2
+      - uses: game-ci/unity-test-runner@v2.0-alpha-5
         id: testRunner
         with:
-          projectPath: test-projects/test-project
-          testMode: all
+          testMode: ${{ matrix.testMode }}
+          checkName: ${{ matrix.testMode }} test results
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
 
```

Comment:
I don't know if we want to go into github tokens here. As I understand it the check doesnt work without them?

Reply:
Yes, providing the token allows for the github checks api to be used.